shared/test-helpers/ganache-cli.sh

```diff
@@ -1,50 +1,64 @@
 #!/usr/bin/env bash
 
 # Exit script as soon as a command fails.
-set -o errexit
+set -o errexit -o pipefail
```

Comment:
Not too familiar with this. Why do we need it? I don't see any pipe script.

Reply:
True, it was instinctively

Reply:
Haha, cool, I think it's ok to leave it anyway if you want (just in case), I just wanted to make sure I was not missing anything ;-)